src/test/jtx/impl/Env.cpp

```diff
@@ -503,7 +503,8 @@ Env::autofill(JTx& jt)
     }
     catch (parse_error const&)
     {
-        test.log << "parse failed:\n" << pretty(jv) << std::endl;
+        if (!parseFailureExpected_)
```

Comment:
What is the objective of this change? Just making it easier to identify errors?

Reply:
It add too much of  false-positive unnecessary output to UT logs which complicates looking for real problem.

Reply:
Should there be additional `BEAST_EXPECT` logic to ensure that an error is actually thrown here?

Reply:
Line 504 is the original line, it didn't changed

Reply:
I understand that. I feel something like the below snippet is more informative:
```
    catch (parse_error const& p_error)
    {
        if (!parseFailureExpected_)
           test.log << "parse failed:\n" << pretty(jv) << std::endl;
           test.log << "Exception thrown: " << p_error << std::endl;
    }
```